### ISSUE TITLE
Intel Advisor RPM issue

### ISSUE THREAD

**Issue by shahzebsiddiqui** [hpcugent/easybuild-easyconfigs#4443]
I was trying to install Advisor and I ran into cpio error. Is there any way to remove the prelink for ld during the build.

See https://access.redhat.com/solutions/148653
 
```
Error unpacking rpm package Advisor-2017_update1-eb_3.1.2-2.x86_64
error: unpacking of archive failed on file /nfs/grid/software/RHEL7/easybuild/software/Core/Advisor/2017_update1/advisor_2017.1.1.486553/lib32/pinruntime/glibc/ld-2.3.4.so;58e29eb0: cpio: Digest mismatch

```

**Comment by boegel** [comment]
@shahzebsiddiqui This is an FPM problem, not an EasyBuild problem?

**Comment by shahzebsiddiqui** [comment]
Yes, it is a FPM problem. From my experience with RPM, the solution was to remove the prelink from the shared library in this case explicitly define this in spec file or do this in easyblock to remove the link using prelink -u <library>

This post will give more insight on how to deal with this issue: https://github.com/jordansissel/fpm/issues/262

**Comment by akesandgren** [comment]
Did not have any problems installing Advisor/2017_update1 on our Ubuntu system. It should be installing from a tar file when using EB.

**Comment by shahzebsiddiqui** [comment]
I am trying to install the RPM generated from EB for our prod systems. EB installs Advisor and generates the RPM without any problem it is only when I try using yum this causes an issue. We are running RHEL 7.2, is it possible someone can try this on a centos or RHEL7 machine if possible.

I think if we remove the prelink for libc in Advisor then the rpm will install. I know how to do this manually but I don't know how to do this via EB.

**Comment by shahzebsiddiqui** [comment]
Can I please get an update on this issue. The RPM is not installing for Advisor. Can we fix this somehow in easyblock
